Source/WebGPU/WebGPU/Queue.mm

```diff
@@ -278,14 +278,17 @@
     if (!(buffer.usage() & WGPUBufferUsage_CopyDst))
         return false;
 
+    if (!buffer.buffer().contents)
```

Comment:
I think this check might be problematic.

Reply:
I think a private buffer which we create in some cases will have a nullptr contents so this check is only valid if `buffer.buffer().storageMode != MTLStorageModePrivate`